src/widgets/DiagramWidget.tsx

```diff
@@ -667,6 +673,12 @@ export class DiagramWidget extends BaseWidget<DiagramProps, DiagramState> {
 		);
 	}
 
+	preventPageZoom(event) {
+		if (event.ctrlKey) {
```

Comment:
ctrlKey? Is this true when you're pinch zooming?

Reply:
Yep, was a bit surprised too : )